tensorrt_llm/_torch/modules/linear.py

```
@@ -1355,6 +1367,7 @@ def load_kv_scales(self, weights: List[Dict]):
                 v_scale.append(w["v_scale"][...].reshape([]))
         return k_scale, v_scale
 
+    #NVFP4 一
```

Flake8 (7.3.0)

[error] block comment should start with '# '

(E265)